Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -393,14 +393,14 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
  *
  * @return Unmutable array of _CTLine objects created with the requested parameters.
  */
-static _CTLine* _DWriteGetLine(CFAttributedStringRef string) {
+static CTLineRef _DWriteGetLine(CFAttributedStringRef string) {
```

Comment:
I'm not sure why this one is Get since it returns a +1, but can you change this to Create*